install/ansible/istio/tasks/determine_version.yml

```diff
@@ -3,6 +3,20 @@
     istio_version_to_use: "{{ istio.release_tag_name }}"
   when: istio.release_tag_name != ''
 
+- name: Set non-empty Authorization header dictionary
+  set_fact:
+    github_api_authorization_header:
+      Authorization: "{{'token ' + github_api_token}}"
+  when:  github_api_token != ''
+
+- name: Set empty Authorization header dictionary
+  set_fact:
+    github_api_authorization_header: {}
+  when:  github_api_token == ''
+
+- name: Set HTTP headers to use
+  set_fact:
+    github_api_http_headers: "{{ default_github_api_headers | combine(github_api_authorization_header)}}"
```

Comment:
where is default_github_api_headers defined?

Reply:
Since it's a variable that is not meant to be changed by whoever executes the playbook, it's defined in `install/ansible/istio/vars/main.yml`

Reply:
right - I see the definition now.  Thanks.  Not sure how I missed it originally.